Round_1_query_6.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
+100/(COUNT(listing_id))*
+(SELECT
+COUNT(listing_id)
+FROM Listings
+WHERE amenities like '%wifi%') as percent_of_wifi
+FROM Listings
```

Comment:
This query (even if it works) will not return a single number.

Reply:
It hasn't completed yet